types/africastalking/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": true,
+        "strictFunctionTypes": true,
+        "baseUrl": "../",
+        "typeRoots": [
+            "../"
+        ],
+        "types": [],
+        "noEmit": true,
+        "forceConsistentCasingInFileNames": true,
+        "esModuleInterop": true
```

Comment:
This should be disabled; see the bot's comment.

Reply:
already disabled them @jakebailey

Reply:
The file doesn't parse anymore. Can't have trailing commas in these files; DT is stricter here.

Reply:
already resolved..hopefully its now resolved

Reply:
unfortunately I cant seem to find the bots comment regarding what I should disable..for clarity purposes what in particular config was I to disable?